pkg/apis/kops/v1alpha2/bastion.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package v1alpha2
 
 type BastionSpec struct {
-	BastionPublicName string `json:"bastionPublicName,omitempty"`
+	PublicName string `json:"bastionPublicName,omitempty"`
```

Comment:
I was just thinking about how you were doing version conversions.  This is a really clever trick :+1: